rollingpin/deploy.py

```diff
@@ -56,14 +56,14 @@ class Deployer(object):
     def __init__(self, config, event_bus, parallel,
                  sleeptime, timeout, dangerously_fast):
         """
-        :param config: (dict)
-        :param event_bus: (EventBus)
-        :param parallel (int)
-        :param sleeptime (float)
-        :param timeout (int)
-        :param dangerously_fast (bool)
-        """
+        :param: config: (dict)
+        :param: event_bus: (EventBus)
+        :param: parallel (int) number of hosts to process in parallel
```

Comment:
Is this right?  I just meant you have:

```
:param event_bus:
```

And then:
```
:param parallel
```

i.e. a colon after the argument in the first one and none on the second.  If you're trying to follow the Sphinx format, I think your original way was correct -- it just hadn't been applied the same to the last 4.